packages/manager/apps/dedicated/client/app/dedicated/server/dns/add/dedicated-server-dns-add.html

```diff
@@ -50,7 +50,7 @@
                             id="domain"
                             name="domain"
                             data-ng-model="entry.domain"
-                            data-ng-pattern="/.+\..+/"
+                            data-ng-pattern="/^[^<>]+\.[^<>]+$/"
```

Comment:
No global options to forbid this ? Maybe some sort of decorator could have been helpful

Reply:
This one is specific, as it is adapted for domain input. Do you mean why not creating a decorator for other ones? (For ng-pattern="/^[^<>]+$/").
I can create it, but it implies to inject in all the concerned controllers scope, what do you think?